app/javascript/src/components/Editor.tsx

```diff
@@ -19,6 +20,11 @@ import './FlowChart.css'
 import { StateConsumer, ApplicationState } from '../Store'
 import { save } from '../persistance'
 import { clone } from '../clone'
+import * as _ from 'lodash'
+import { link } from 'fs'
```

Comment:
Maybe I'm misreading, but should this be importing from 'fs'? At first glance it looks like maybe your editor auto-imported this inappropriately; I don't see it used in the file.